servers/rendering/renderer_rd/renderer_scene_render_rd.h

```diff
@@ -204,9 +204,9 @@ class RendererSceneRenderRD : public RendererSceneRender {
 
 	/* LIGHTING */
 
-	virtual void setup_added_reflection_probe(const Transform3D &p_transform, const Vector3 &p_half_extents){};
+	virtual void setup_added_reflection_probe(const Transform3D &p_transform, const Vector3 &p_half_size){};
 	virtual void setup_added_light(const RS::LightType p_type, const Transform3D &p_transform, float p_radius, float p_spot_aperture){};
-	virtual void setup_added_decal(const Transform3D &p_transform, const Vector3 &p_half_extents){};
+	virtual void setup_added_decal(const Transform3D &p_transform, const Vector3 &p_half_size){};
```

Comment:
Note that `p_half_extents` was actually named wrong before. The `extents` (unchanged) were always passed in, which is half the size.